### PR TITLE
added icons for navigation

### PR DESCRIPTION
**Pull request by wakooka**

### CODE REVIEW

**Comment by Bidjii** [comment]
Changelog

**Comment by wakooka** [comment]
[
capture d ecran 2018-08-23 a 11 49 36
](url)

**Comment by loick** [comment]
While you're on the icons and the generated files, can you add in the template, a comment to specify which files are actually generated? 

And we should create another ticket to build the generated files instead of commiting it.

File: src/icon/index.tsx [hunk 1]
```diff
@@ -1,65 +1,81 @@
...
+  BankIcon,
+  BellIcon,
   CheckIcon,
   ChevronIcon,
   CircleIcon,
   ClockIcon,
+  CoinIcon,
   ComfortIcon,
   CrossIcon,
+  CrossdiskIcon,
```
Comment (Tuxosaurus) [inline src/icon/index.tsx]: CrossDiscIcon

File: src/icon/ticketIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        viewBox="0 0 24 15"
+        xmlns="http://www.w3.org/2000/svg"
+        className={cc(['kirk-icon', className])}
+        width={size}
+        height={size}
+        aria-hidden={isEmpty(title)}
+        fill={iconColor}
+      >
+        { title && <title>{title}</title> }
+        <path fill-rule="nonzero" d="M20.5 7.75c0 1.1 1.1 2.03 2.5 2.03.28 0 .5.23.5.5v4.22a.5.5 0 0 1-.5.5H1a.5.5 0 0 1-.5-.5v-4.22c0-.27.22-.5.5-.5 1.4 0 2.5-.93 2.5-2.03 0-1.1-1.1-2.03-2.5-2.03a.5.5 0 0 1-.5-.5V1C.5.72.72.5 1 .5h22c.28 0 .5.22.5.5v4.22a.5.5 0 0 1-.5.5c-1.4 0-2.5.93-2.5 2.03zm-1 0c0-1.54 1.32-2.8 3-3V1.5h-21v3.25c1.68.2 3 1.46 3 3s-1.32 2.8-3 3V14h21v-3.25c-1.68-.2-3-1.46-3-3z"/>
```
Comment (Bidjii) [inline src/icon/ticketIcon.tsx]: `fill-rule` > `fillRule`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Unreleased
-- [...]
+- **[NEW]** Icons for Navigation (BankIcon, BellIcon, BubbleIcon, CoinIcon, CrossdiskIcon, HomeIcon, NewspaperIcon, PlusIcon, ProfileIcon, TicketIcon)
```
Comment (Bidjii) [inline CHANGELOG.md]: `CrossdiskIcon` > `CrossdiscIcon`

Comment (Bidjii) [reply CHANGELOG.md]: And the `PlusIcon` is not really a new one :p

**Pull request merged**